content/applications/websites/ecommerce/checkout_payment_shipping/shipping.rst

```diff
@@ -91,17 +91,50 @@ if the shiping method does not involve any specific provider.
      based on the customer's area;
    - click the :guilabel:`Test Environment` smart button to switch to
      the :guilabel:`Production Environment`, then click :guilabel:`Unpublished` to
-     :guilabel:`Publish` the shipping method and make it available to website visitors.
-
-.. _ecommerce/shipping/instore-pickup:
-
-In-store pickup
-===============
-
-To allow customers to reserve products online and pay for/collect them in person at the store, go to
-:menuselection:`Website --> Configuration --> Settings`, scroll to the :guilabel:`Shipping` section,
-enable :guilabel:`On Site Payments & Picking`, and :guilabel:`Save`.
-
-Then, click :guilabel:`Customize Pickup Sites`, select the shipping method or click :guilabel:`New`
-to create a new one and :ref:`configure <inventory/shipping_receiving/configure-delivery-method>`
-the fields. Make sure the :guilabel:`Provider` field is set to :guilabel:`Pickup in store`.
+     :guilabel:`Publish` the delivery method and make it available to website visitors.
+
+.. _ecommerce/delivery/instore-pickup:
+
+Pick up in store
+================
+
+To allow customers to reserve products online and pay for/collect them in-store, go to
+:menuselection:`Website --> Configuration --> Settings`. Scroll to the :guilabel:`Delivery` section,
+enable :guilabel:`Click & Collect`, and :guilabel:`Save`.
+
+Next, click :icon:`fa-arrow-right` :guilabel:`Configure Pickup Locations` to
+:ref:`set up <inventory/shipping_receiving/configure-delivery-method>` your delivery method.
+Ensure the :guilabel:`Provider` field is set to :guilabel:`Pickup in store`. Under the
+:guilabel:`Stores` tab, click :guilabel:`Add a line` to add the :guilabel:`Warehouse` and its
+:guilabel:`Address` where customers can collect their orders, and the :guilabel:`Opening hours`.
+
+Once your set up is complete, click the :guilabel:`Unpublish` button to change the status to
+:guilabel:`Publish`, making the delivery method available to customers. When the product is in
+stock, a **location selector** is displayed on :doc:`product <../products>` and
+:doc:`checkout <checkout>` pages.
+
+.. note::
+   - Each warehouse must have a **complete address** to ensure its location is accurately displayed to
+     customers. Incomplete addresses prevents the warehouse from being shown.
+   - **Services** are not available for the pick up in store option.
+   - Customers cannot select a pick-up location if the product is out of stock at that location. The
+     :ref:`Continue selling <ecommerce/products/stock-management>` option for out-of-stock products
+     is not supported.
+   - If the :ref:`Show Available Qty <ecommerce/products/stock-management>` option is enabled for a
+     product, customers can view the stock quantity available for each warehouse in the location
+     selector on the product page.
+   - Customers cannot complete the :doc:`checkout <checkout>` if the selected products are out of
+     stock at the chosen pick-up location.
```

Comment:
On line 120 you say that if the product is out of stock, you can't select a location so I'm not sure I understand how people would even come close to completing the checkout process if they can't select a location in the first place? This sounds very similar to the list item on line 120 and if they're not the same thing, I think we should clarify how/why (and maybe group the two list items together to avoid repetitions).